day7/will/index.css

```diff
@@ -0,0 +1,682 @@
+    font-size: 20px;
+    font-weight: 500;
+    color: white;
+    line-height: 24px;
+    padding: 22px 40px;
+    border-radius: 50px;
+    background-color: #ea3f58;
+    border: none;
+    position: absolute;
+    bottom: 40px;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .mobileview-wrapper {
+    display: none;
+  }
+
+  .laptop-wrapper {
+    display: none;
+  }
+
+  .tabletview-wrapper {
+    position: relative;
+    display: block;
+  }
+
+  .tabletview-title-container img {
+    width: 15%;
+    margin-top: 50px;
+    margin-left: 30px;
+  }
+
+  .tabletview-title-container {
+    z-index: 1;
+    width: 100%;
+    position: fixed;
+    top: 0px;
+    background: linear-gradient(0deg, #6666ff 0, #002b80 100%);
+  }
+
+  .tabletview-title-block {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-left: 0px;
+  }
+
+  .tabletview-title-circle {
+    width: 220px;
+    height: 220px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    border-radius: 50%;
+    border: 8px solid rgba(255, 255, 255, 0.1);
+    color: white;
+    margin: 40px 50px 50px 0px;
+    background: rgba(0, 0, 0, 0.1);
+    box-shadow: 0px 0px 0px 4px rgba(0, 0, 0, 0.1);
+  }
+
+  .tabletview-title-circle p {
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 20px;
+  }
+
+  .tabletview-title-circle h2 {
+    font-weight: bold;
+    font-size: 48px;
+    line-height: 74px;
+    margin: 0px;
+  }
+
+  .tabletview-title-circle h4 {
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 20px;
+    color: #ed882c;
+  }
+
+  .tabletview-title-text h3 {
+    font-weight: bold;
+    font-size: 24px;
+    line-height: 24px;
+    color: #fff;
+  }
+
+  .tabletview-title-text p {
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 20px;
+    color: rgba(255, 255, 255, 0.5);
+  }
+
+  .tabletview-title-progress p {
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 17px;
+    color: #fff;
+    padding-right: 50px;
+  }
+
+  .tabletview-title-progress {
+    background-color: rgba(255, 255, 255, 0.1);
+    border-radius: 8px;
+    padding: 15px 30px 30px 30px;
+  }
+
+  .tabletview-progressbar {
+    background-color: rgba(255, 255, 255, 0.2);
+    border-radius: 2px;
+    width: 100%;
+    height: 4px;
+    margin-top: 20px;
+  }
+
+  .tabletview-progressbar div {
+    background-color: #fff;
+    border-radius: 2px;
+    width: 20%;
+    height: 4px;
+  }
+
+  .tabletview-content-container {
+    z-index: 0;
+    position: absolute;
+    width: 90%;
+    top: 410px;
+    margin: 20px 50px 0px 50px;
+  }
+
+  .tabletview-content-title {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+  }
+
+  .tabletview-content-title-text p {
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 17px;
+    color: #000;
+    opacity: 0.5;
```

Comment:
can we use a color with alpha value, instead of a opacity?

Reply:
adjusted